docs/reference/docs/delete-by-query.asciidoc

```diff
@@ -233,26 +233,49 @@ The JSON response looks like this:
 
 The number of milliseconds from start to end of the whole operation.
 
+`timed_out`::
+
+This flag is set to `true` if any of the requests executed during the
+delete by query execution has timed out.
+
+`total`::
+
+The number of documents that were successfully processed.
+
 `deleted`::
 
 The number of documents that were successfully deleted.
 
 `batches`::
 
-The number of scroll responses pulled back by the the delete by query.
+The number of scroll responses pulled back by the delete by query.
 
 `version_conflicts`::
 
 The number of version conflicts that the delete by query hit.
 
+`noops`::
+
+The number of documents that were ignored because the script used for
```

Comment:
I don't believe delete by query supports scripts actually. This is probably just always going to be 0.

This is there because delete-by-query, reindex, and update-by-query always return the same results to keep me sane.

We can probably fix this, but I don't know that it is worth it.

Reply:
Right, thanks